### ISSUE TITLE
vac doesn't work

### ISSUE THREAD

**Issue by Truelime39874** [ValveSoftware/Source-1-Games#5700]
hey so vac doesn't work and there is like cheaters and bots in almost every server would yall mind like fixing that

**Comment by Jeffonehundred** [comment]
They do not care about the cheaters and bots. Just look at @wgetJane that is the type of response you'll get from valve. They just don't care.

**Comment by JoriKos** [comment]
> They do not care about the cheaters and bots. Just look at wgetJane that is the type of response you'll get from valve. They just don't care.

This github is for bugs and technical issues with Source 1 games, simply going "VAC doesn't work" is not an actual technical issue or bug with any Source 1 game and it's not even true. Making these kinds of posts just gives extra work to kisak to remove them every time.

**Comment by JoriKos** [comment]
> They _**do**_ care but they quite literally cant do anything about it with the current amount of employees they have. Valve isn't "lazy" or "giving up", they just don't have the people who can do it. And its not an easy solution to fix either.

This is not true, they have an anti-cheat team, and they definitely have enough people to support it. TF2 is just a low priority for them, so it doesn't get (m)any updates to its anti-cheat.

**Comment by Tiagoquix** [comment]
Possible _duplicate_ of https://github.com/ValveSoftware/Source-1-Games/issues/3477
